src/main/java/org/reaktivity/nukleus/http_cache/internal/HttpCacheConfiguration.java

```diff
@@ -37,6 +38,7 @@
         HTTP_CACHE_SLOT_CAPACITY = config.property("slot.capacity", 0x4000); // ALSO is max header size
         HTTP_CACHE_MAXIMUM_REQUESTS = config.property("maximum.requests", 64 * 1024);
         HTTP_CACHE_ETAG_PREFIX = config.property("etag.prefix", new Random().nextInt(99999));
+        HTTP_CACHE_ALLOWED_CACHE_PERCENTAGE = config.property("allowed.cache.percentage", 95);
```

Comment:
This configuration name is tricky because it means not using the cache for new requests beyond the allowed percentage, even though it is okay to keep using the cache for responses already requested.

Reply:
What do you suggest?!